src/listeners/onload.js

```diff
@@ -1,9 +1,17 @@
-import { renderQuestions } from "../handlers/render-questions.js";
-import { createQuestionList } from "../views/questions-list.js";
+import { renderQuestions } from '../handlers/render-questions.js';
+import { createQuestionList } from '../views/questions-list.js';
 
 window.addEventListener('DOMContentLoaded', () => {
-    const ui = document.getElementById('user-interface');
-    const list = createQuestionList();
+  const ui = document.getElementById('user-interface');
+  const list = createQuestionList();
 
-    renderQuestions(ui, list);
+  renderQuestions(ui, list);
+});
+
+import { createCounts } from '../views/show-amounts.js';
+window.addEventListener('DOMContentLoaded', () => {
+  const ui = document.getElementById('user-interface');
+  const count = createCounts();
+
+  renderQuestions(ui, count);
```

Comment:
Move all of this into a function in handlers/ and then call all the needed handlers without parameters.